docs/dashboard.rst

```diff
@@ -0,0 +1,28 @@
+.. _dashboard:
+
+=========================================================
+Rackspace Technology customer portal solutions dashboard
+=========================================================
+
+Rackspace customers with multiple accounts can use the
+solutions dashboard to view a snapshot of their
+account activity.
```

Comment:
I don't think I'd call this "activity" since it's really just a list of accounts. How about something along the lines of:

> Rackspace customers can use the Solutions Dashboard (pending name decision from Guy/Lan) to see all their accounts across all infrastructure platforms provided by Rackspace.

I'm sure the verbiage is terrible, but I do think it gives a more accurate picture of what its function is. In the future, I'd definitely love to get more into the "snapshot" features, but they don't exist now.

Reply:
yeah, def lose the activity - we don't show activity of any form (yet). 

Also, I'm def no expert and conscious this may lead to bikeshedding, so leave if this is the standard way we do things.... but it feels more natural to me if "[feature name]" is the subject of the intro sentence

E.g. "[Account Hub] shows customers all their accounts in one place..."

Not particularly opinionated though - defer to profession writers' opinions.